test/semantics/test_errors.sh

```diff
@@ -18,7 +18,7 @@
 
 PATH=/usr/bin:/bin
 srcdir=$(dirname $0)
-CMD="${F18:-../../../tools/f18/f18} -fdebug-resolve-names -fparse-only"
+CMD="${F18:-../../../tools/f18/f18} -fdebug-resolve-names -fparse-only -Mstandard"
```

Comment:
Does this do anything useful? It turns on some warnings, but we don't check warnings.

Anyway, if it's going to be the default it can be removed from dosemantics03.f90.

Reply:
Removed.